bdikit/visualization/mappings.py

```diff
@@ -29,7 +29,7 @@ def plot_value_mappings(value_mappings, include_unmatches=True):
 
         if include_unmatches:
             for unmatch_value in value_mappings[column_name]["unmatch_values"]:
-                matches.append((unmatch_value, "-", "-"))
+                matches.append((unmatch_value, "No match found", "-"))
```

Comment:
If we are using natural language, I think we should add something like square brackets around the text to differentiate it from other values: `[No match found]`.

Reply:
Good idea!